.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,67 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.database }}, ${{ matrix.phpcr-transport }})"
+        runs-on: ubuntu-18.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '5.5'
+                      database: mysql
+                      phpcr-transport: doctrinedbal
```

Comment:
would not configure this in the matrix because we dont need to change it when testing the bundle